### ISSUE TITLE
[Quest] Breaking Down Netherock

### ISSUE THREAD

**Issue by Greywolf59** [WoWManiaUK/Redemption#5744]
**Links:**
Quest:
https://tbc.wowhead.com/quest=10701/breaking-down-netherock#comments
https://www.wow-mania.com/armory/?quest=10701
Netherock:
https://tbc.wowhead.com/npc=20772/netherock
https://www.wow-mania.com/armory/?npc=20772
Netherock Crumblet (spawns):
https://tbc.wowhead.com/npc=21323/netherock-crumbler
https://www.wow-mania.com/armory/?npc=21323

**What is Happening:**
The Netherock does not spawn little adds (at least haven't noticed it)

**What Should happen:**
Netherock should spawn adds as you damage him.

**Comment by Rushor** [comment]
indeed, i checked in db, the creatures does not have an AI / script at all.

all implemented now according to wowhead. Adds will spawn at 75, 50 and 25 percent.

**Comment by Echinaceax** [comment]
confirmed fixed in PTR 👍 

![image](https://user-images.githubusercontent.com/39439201/156934376-cdf29328-e973-41ed-baa2-3c008170252c.png)
